aks/etcd-operator.yaml

```diff
@@ -0,0 +1,17 @@
+# TODO: Move me to helm chart when we can sort ordering:
```

Comment:
There is ordering issues with CRD and HELM. 
In addition, if you delete/recreate CRD operators will kill etcd. You can delete operator, but you cant delete crd. 
Maybe we could maintain things like this with ROOT level chart for all HCP?